tests/ui_tools/test_buttons.py

```diff
@@ -422,6 +431,58 @@ def test__decode_stream_data(
 
         assert return_value == expected_response
 
+    @pytest.mark.parametrize(
+        "pm_data, expected_response",
+        [
+            case(
+                "1001,12-pm",
+                dict(type=None, recipient_ids=[1001, 12], recipient_emails=None),
```

Comment:
Since you type these `expected_response` cases as `DecodedPM` in the test parameters below, it would be more appropriate to have these as `DecodedPM` instances rather than `dict` instances (similar to how we have `ParsedNarrowLink` instances as parameter cases in the other tests in this file).